springfield/firefox/templates/firefox/browsers/mobile/index.html

```diff
@@ -0,0 +1,173 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros.html" import google_play_button, fxa_email_form with context %}
+{% from "macros-protocol.html" import split with context %}
+
+{% extends "base-protocol.html" %}
+
+{% set show_firefox_app_store_banner = switch('firefox-app-store-banner') %}
+{% set android_url = play_store_url('firefox', 'firefox-browsers-mobile') %}
+{% set ios_url = app_store_url('firefox', 'firefox-browsers-mobile') %}
+{% set referrals = '?utm_source=www.firefox.com&utm_medium=referral&utm_campaign=firefox-mobile' %}
+{% set _entrypoint = 'mozilla.org-firefox-mobile' %}
```

Comment:
update entrypoint too?

```suggestion
{% set _entrypoint = 'firefox.com-firefox-mobile' %}
```